### DISCUSSION TITLE
Problem - Exceeds Number of Requests

### DISCUSSION THREAD

**Discussion by 290989** [Hyundai-Kia-Connect/kia_uvo#923] [General]
Hi, 
Since a few days I have the following problem often:

The integration isn‘t available & I see the following information:

![image](https://github.com/user-attachments/assets/78914b31-1421-4cfc-b84e-546691ac5cb2)

How I can find the problem? How can I see, what give a request to the Kia etc?

I am using the most time Home Assistant. Sometimes the original Kia app (but not often)

That’s my config:
![image](https://github.com/user-attachments/assets/026a0aff-11b6-4f13-ad5e-f2444faf9097)

thank you so much,
Regards Daniel

**Comment by cdnninja** [comment]
Try drop your scan interval to 60.
